src/main/java/gregtech/common/metatileentities/multi/MetaTileEntityFissionReactor.java

```diff
@@ -113,27 +115,33 @@ public int findHeight(boolean top) {
     protected void updateFormedValid() {
         // Take in coolant, take in fuel, update reactor, output steam
 
-        if (this.locked) {
+        if (this.lockingState == LockingState.LOCKED) {
 
             // Coolant handling
             for (ICoolantHandler coolantImport : this.getAbilities(MultiblockAbility.IMPORT_COOLANT)) {
+                // TODO: Move into coolant import hatch
                 this.fissionReactor.heatRemoved += coolantImport.getCoolant().getProperty(PropertyKey.COOLANT)
                         .getCoolingFactor() * this.flowRate;
                 coolantImport.getFluidTank().drain(this.flowRate, true);
             }
             for (ICoolantHandler coolantExport : this.getAbilities(MultiblockAbility.EXPORT_COOLANT)) {
+                // TODO: Move into coolant export hatch
                 coolantExport.getFluidTank().fill(coolantExport.getCoolant().getProperty(PropertyKey.COOLANT)
                         .getHotHPCoolant().getFluid(this.flowRate), true);
             }
 
             // Fuel handling
             if (this.fissionReactor.fuelDepletion == 1.) {
-                boolean gotFuel = true;
+                boolean hasEmpty = false;
```

Comment:
this varible seems to never be updated

Reply:
Should be fine for now, we're still working on missing stuff on the actual branch, these are just fixes for the major crashes